packages/soci-snapshotter/soci-snapshotter.spec

```diff
@@ -42,6 +42,24 @@ Conflicts: (%{_cross_os}image-feature(no-fips) or %{name}-bin)
 %description fips-bin
 %{summary}.
 
+%package cli-bin
+Summary: soci CLI binary,
```

Comment:
It can be just `Soci CLI`, following what was done for `Docker`. But given the description of the main package, you could do `Remove management agent CLI`